### ISSUE TITLE
Create a production install of Airflow on tigrsrv

### ISSUE THREAD

**Issue by DESm1th** [TIGRLab/air-tigrs#4]
This is _mostly_ done. At this moment if you ssh to tigrsrv and run `sudo systemctl start airflow-webserver` you will be able to access our production instance from any lab computer's browser at `http://tigrsrv.camhres.ca`. The only account currently existing is an admin account with the username localadmin and our old admin password.

However, the reason I say mostly, and the reason why I currently have the systemd module stopped is because there are a few housekeeping tasks I want to work on first.

- [x] Airflow's webserver's log output (browser connections, etc.) gets mixed in with syslog and I want to finish writing rsyslog rules to redirect it to a separate log file because it's pretty spammy and drowns out important system messages
- [x]  Currently nginx is serving airflow over http not https. This is _mostly_ ok since we're on a private network, but I want to take a look at switching to https. Most services to setup https for free, like letsencrypt, require your site to be publicly accessible so I need to find a solution here.
- [x] The airflow-scheduler systemd module that airflow relies on is currently disabled because it spams /var/log/syslog with three messages a second due to the postgresql connections and oidentd needing to fire up. I need to write rsyslog rules to silence this before we start running it

Other 'nice to have' things I still want to work on:
- [ ] Updating airflow's mail configuration so it can send 
emails
- [x] (EDIT: We decided this wasnt needed) Setting up either oauth or (if possible) camh's ipa server for accounts

Overview of Production Server:
- Airflow home is at `/archive/code/airflow`
- Its virtual env is at `/archive/code/virtual_envs/airflow`
- It runs from tigrsrv under user airflow and group airflow. These are locally defined on tigrsrv and nowhere else
- It uses the dashboard's database on srv-postgres. It runs as a schema named 'airflow' within this database. You can access it from any workstation, while logged in as clevis, with `psql -U airflow -h srv-postgres dashboard`. Note that because it's a schema you have to prefix commands with `airflow.`. For instance, to view all tables after connecting you would use `\dt airflow.*` rather than just `\dt`
- Airflow logs will be stored under `/var/log/airflow/`
- It has two systemd modules which must both be running: `airflow-webserver` and `airflow-scheduler`
